### PR TITLE
doc: typo fix

### PR DESCRIPTION
**Pull request by Ashid6**
Debut patch for minimum outreachy requirements. More coming up!

### CODE REVIEW

File: doc/dev/osd_internals/pg_removal.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ that point will be ordered after the pg deletion.
 OSD::_create_lock_pg must handle two cases:
 
   1. Either there is no DeletingStateRef for the pg, or it failed to cancel
-  2. We succeeded in canceling the deletion.
+  2. We succeeded in cacancelling the deletion.
```
Comment (alfredodeza) [inline doc/dev/osd_internals/pg_removal.rst]: s/cacancelling/cancelling/

**Comment by dmick** [comment]
Please fix the title too; I spent far too long trying to figure out what a dashottar was

File: doc/dev/osd_internals/pg_removal.rst (hunk 1 above)
Comment (Ashid6) [reply doc/dev/osd_internals/pg_removal.rst]: + cancelling

**Comment by Ashid6** [comment]
is this an ARM specific issue?

**Comment by tchaikov** [comment]
@Ashid6 i guess you replied to a wrong PR?

**Comment by Ashid6** [comment]
my build broke when it came to arm 64. I forked the ceph repo.

**Comment by Ashid6** [comment]
These two tests failed at build time:
1 - run-rbd-unit-tests.sh (Failed)
110 - unittest_journal (OTHER_FAULT)

Any idea why? As my change list just comprising of typo fixes.

**Comment by tchaikov** [comment]
@Ashid6 the "make check" failures on arm64 are known ones. they are not relevant to your change. also, could you remove the merge commit from your PR?

**Pull request merged**